### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by Swapna804**

### CODE REVIEW

**Review by Manvityagi — CHANGES_REQUESTED** [review]
@Swapna804 Some of your files are in the root folder. All your commits should only be in your name folder. And there should be segregations of milestone folders inside your name folder too.

File: Swapna Devi/milestone1/3.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    if(n%2==0)
+    {
+	cout<<n<<" is even";
+    }
+    else
+    {
+	cout<<n<<" is odd";
+    }
+    return 0;
+}
```
Comment (Manvityagi) [inline Swapna Devi/milestone1/3.cpp:18]: all these under Swapna Devi/milestone1 are fine and good to be merged

File: 3.cpp [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//Write a program to check whether a number is even or odd
+#include<iostream>
+using namespace std;
+int main()
+{
```
Comment (Manvityagi) [inline 3.cpp]: All these 1.cpp,2.cpp,3.cpp etc needs to be deleted as well.

File: Basics/3.cpp [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    if(n%2==0)
+    {
+	cout<<n<<" is even";
+    }
+    else
+    {
+	cout<<n<<" is odd";
+    }
+    return 0;
+}
```
Comment (Manvityagi) [inline Basics/3.cpp]: All these Basics/1.cpp,Basics/2.cpp etc needs to be deleted as well before approval for merging.

**Review by Manvityagi — CHANGES_REQUESTED** [review]

File: 1.cpp [hunk 4]
```diff
@@ -0,0 +1,7 @@
+//Write a program to print  “Welcome to Girl Code It”?
+#include<iostream>
+using namespace std;
+int main()
+{
+    cout<<"Welcome to Girl Code It"<<endl;
+}
```
Comment (Manvityagi) [inline 1.cpp]: delete these as well. @Swapna804

**Review by Manvityagi — CHANGES_REQUESTED** [review]

File: Basics/3.cpp (hunk 3 above)
Comment (Swapna804) [reply Basics/3.cpp]: i have deleted the basics and added milestone folder under my name folder .

File: 1.cpp (hunk 4 above)
Comment (Swapna804) [reply 1.cpp]: Could you please check. I  am new to open source so i am not sure

Comment (Manvityagi) [reply 1.cpp]: There are still 5 extra files in the root folder which are outside your name folder. Delete them too and your PR will be ready to merge 😄

Comment (Swapna804) [reply 1.cpp]: how to know where these files are

**Review by Manvityagi — APPROVED** [review]
Good to go now :)

**Comment by Swapna804** [comment]
Thankyou maam. Could i continue with other milestones

**Pull request merged**